pkg/controllers/monitoring/monitoring-stack/controller.go

```diff
@@ -144,11 +147,40 @@ func (rm resourceManager) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return ctrl.Result{}, nil
 	}
 
+	// Check if the plugin is being deleted
 	if !ms.ObjectMeta.DeletionTimestamp.IsZero() {
-		logger.V(6).Info("skipping reconcile since object is already schedule for deletion")
+		logger.V(6).Info("removing cluster scoped resources")
+
+		reconcilers := stackComponentCleanup(ms)
```

Comment:
just a nit over here, if we have the prometheus and the alertamanager names, why not adding that also as a log? It may be useful when debugging. Otherwise lgtm!

Reply:
This is just an info log. In line 158 an error is logged in case something goes wrong, that'll include the resource name that creates an error. Is that what you're after?

Not sure we should log 6 resource names on the happy path.

Reply:
nvm, mistook the 'Info' for 'Err' in my mind, LGTM then, thanks!